eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -49,40 +49,80 @@ param(
   [string]$PRBody = $PRTitle,
 
   [Parameter(Mandatory = $false)]
-  [string]$PRLabels
+  [string]$PRLabels,
+
+  [Parameter(Mandatory = $false)]
+  [string]$UserReviewers,
+
+  [Parameter(Mandatory = $false)]
+  [string]$TeamReviewers,
+
+  [Parameter(Mandatory = $false)]
+  [string]$Assignees
 )
 
+. "${PSScriptRoot}\logging.ps1"
+
 $headers = @{
   Authorization = "bearer $AuthToken"
 }
+$baseURI = "https://api.github.com/repos"
+function SplitMembers ($membersString)
+{
+  return @($membersString.Split(",") | % { $_.Trim() } | ? { return $_ })
```

Comment:
I'd suggest adding a check for empty like:
```
if (!$memberString) { return $null }
```

That prevents any cases were we have an empty/null value and turn it into and empty array.